spigot/src/main/kotlin/com/mineinabyss/geary/minecraft/actions/ApplyPotionAction.kt

```diff
@@ -0,0 +1,25 @@
+package com.mineinabyss.geary.minecraft.actions
+
+import com.mineinabyss.geary.ecs.GearyEntity
+import com.mineinabyss.geary.ecs.actions.GearyAction
+import com.mineinabyss.geary.minecraft.components.toBukkit
+import com.mineinabyss.idofront.serialization.PotionEffectSerializer
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+import org.bukkit.entity.LivingEntity
+import org.bukkit.potion.PotionEffect
+import kotlin.random.Random
+
+@Serializable
+@SerialName("geary:apply_potion_effects")
+public data class ApplyPotionAction(
```

Comment:
I'd like to start documenting all our actions, components, etc... from now on since I plan to eventually generate a documentation page that would explain all the parameters and whatnot for end users.